### PR TITLE
Add note to .analysis_options about .analysis_options in flutter/plugins

### PR DESCRIPTION
**Pull request by goderbauer**

### CODE REVIEW

**Review by tvolkert — APPROVED** [review]

File: .analysis_options [hunk 1]
```diff
@@ -16,6 +16,9 @@
 # It is very similar to the .analysis_options_repo file in this same directory;
 # the only difference (currently) is the public_member_api_docs option,
 # which triggers too many messages to be used in editors.
+#
+# NOTE: Please keep this file in sync with
```
Comment (Hixie) [inline .analysis_options]: s/NOTE// to follow our style guide. :-)

Comment (Hixie) [inline .analysis_options]: ...but even better would be to edit the earlier text to refer to this file organically along with the other three files.

Comment (goderbauer) [inline .analysis_options]: Done. PTAL.

**Comment by Hixie** [comment]
LGTM

**Comment by a14n** [comment]
LGTM

**Pull request merged**